src/OrchardCore.Modules/OrchardCore.Users/Views/Admin/EditPassword.cshtml

```diff
@@ -1,10 +1,18 @@
-@model OrchardCore.Users.ViewModels.ResetPasswordViewModel
+@model OrchardCore.Users.ViewModels.EditPasswordViewModel
 
-<zone Name="Title"><h1>@RenderTitleSegments(T["Edit Password"])</h1></zone>
+<zone Name="Title"><h1>@RenderTitleSegments(T["Change Password"])</h1></zone>
 
 <form asp-controller="Admin" asp-action="EditPassword" method="post" class="form-horizontal no-multisubmit">
     <div asp-validation-summary="All"></div>
     <input asp-for="UsernameOrEmail" type="hidden" />
+    <div class="mb-3">
+        <label asp-for="CurrentPassword" class="col-md-4 form-label">@T["Your Current password"]</label>
```

Comment:
Hum, I think we also need to account for users with no password. The local user could have not password "user using external authentication".

Reply:
We could have a look at this in another PR. Coz it's not handled in the current code-base IMHO